test/model/src/driver/kv_workload_generator.cpp

```diff
@@ -46,8 +46,12 @@ kv_workload_generator_spec::kv_workload_generator_spec()
     max_sequences = 1000;
     max_concurrent_transactions = 3;
 
+    max_recno = 10'000;
```

Comment:
I can see where this is used but I don't understand the intent. Could you explain it please? (FWIW, WT's `max_recno` is 2^64, so this is quite a restriction.)

Reply:
The main reason why it is constrained this much is because of FLCS. When you create key 10,000, WT would implicitly also create keys 1–9,999 if they don't already exist. When I had it set to 1,000,000, it felt like the tests were taking forever. We don't need to cover the entire key space—at least I'm not particularly aware of anything where this would matter. We could have a higher default limit if we'd like; I did some benchmarking, and 50,000 and 100,000 still look somewhat reasonable.

Reply:
Understood - thanks for the explanation.